vllm/model_executor/layers/quantization/kernels/scaled_mm/__init__.py

```diff
@@ -68,6 +68,13 @@ def choose_scaled_mm_linear_kernel(
             )
             continue
 
+        # Check if kernel is supported on this platform/capability
+        if hasattr(kernel, "is_supported"):
+            supported, reason = kernel.is_supported(compute_capability)
+            if not supported:
+                failure_reasons.append(f" {kernel.__name__}: {reason}")
+                continue
```

Comment:
We should just always use `is_supported` instead of `compute_capability` like done in my draft PR #19434, could you make that change here?